MantidQt/MantidWidgets/src/InstrumentView/InstrumentWidgetPickTab.cpp

```diff
@@ -15,14 +15,19 @@
 #include "MantidGeometry/Instrument/ParameterMap.h"
 #include "MantidAPI/AnalysisDataService.h"
 #include "MantidAPI/Axis.h"
-#include "MantidAPI/ITableWorkspace.h"
+#include "MantidAPI/FrameworkManager.h"
 #include "MantidAPI/IPeaksWorkspace.h"
+#include "MantidAPI/ITableWorkspace.h"
 #include "MantidAPI/MatrixWorkspace.h"
 #include "MantidAPI/Sample.h"
 #include "MantidAPI/WorkspaceFactory.h"
 #include "MantidAPI/TableRow.h"
-#include "MantidAPI/FrameworkManager.h"
+#include "MantidAPI/WorkspaceFactory.h"
```

Comment:
Do we need all of these extra includes? I can't see where things like `ConfigService`, `WorkspaceFactory`, `ParameterMap.h` are needed?